coalib/files/Fileproxy.py

```diff
@@ -0,0 +1,89 @@
+from coala_utils.decorators import generate_eq
```

Comment:
I didnt see an answer to https://github.com/coala/coala/pull/2784#pullrequestreview-5991591 :

Why is this creating coalib/files/__init__.py ; those namespace files are only needed pre 3.4 ?